spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -584,7 +584,7 @@ If a bean explicitly declares a scope, any default scopes declared by stereotype
 
 === Default bean discovery mode
 
-The default _bean discovery mode_ for a bean archive is `annotated`, and such a bean archive is said to be an _implicit bean archive_ as defined in <<bean_archive>>.
+The default _bean discovery mode_ for a bean archive is `annotated`, and such a bean archive is said to be an _implicit bean archive_ as defined in <<bean_archive_full>>.
```

Comment:
This is part of Lite, so should stay pointing to `bean_archive`?

Reply:
In fact, `<<bean_archive>>`  was a pointer to the Full variant even before ;-)
That said, you're right - will fix that.